dali/pipeline/operator/op_spec.h

```diff
@@ -94,6 +94,7 @@ class DLL_PUBLIC OpSpec {
    */
   template <typename T>
   DLL_PUBLIC inline OpSpec& AddArg(const string &name, const T &val) {
+    VerifyRenamedAfterDeprecated(name);
```

Comment:
Why here?
Wont be called anyway in the chain of AddArg->SetArg->SetInitializedArg->VerifyRenamedAfterDeprecated?

Reply:
We need to issue this error before the regular error of duplicated arguments. So I added it in all places where we have the check for duplicated arguments.